docs/foundations/color/theming.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: layout-foundations.njk
+title: Theming
+heading: Color
+tags:
+  - color
+permalink: /foundations/color/theming/index.html
+order: 10
+bodyClasses: element-docs
+---
+
+## Theming
+
+There are various themes within our design system. The themes you will use the 
+most for the majority of your projects are the light and dark themes.
+
+### How themes work
+
+Themes allow for the creation of different experiences using the same design 
+system. When a theme is changed, elements change including color, space, text, 
+and more. Layouts, content, and imagery usually stay the same.
+
+## Choosing a theme
+
+How you choose a theme is based on content, user experience, and accessibility 
+needs.
+
+### Light theme
+
+The light theme is the default theme and has lots of use cases.
+
+{% example palette="light",
+          alt="examples of several elements against a white surface",
+          src="/assets/color/light-theme.png" %}
+
+### Dark theme
+
+The dark theme can be used for highlighting content with dark colors or if a 
+brighter interface would otherwise disrupt the user experience. Most light theme 
+elements and patterns have dark theme counterparts.
+
+{% alert title="Brand red and accessibility",
+            state="warning" %}
+Do not apply the Red Hat red color to text in dark environments unless
+it meets [WCAG 2.1 AA][wcag-2-1-aa] requirements.
+{% endalert %}
+
+{% example palette="darkest",
+          alt="examples of several elements against a dark gray surface",
+          src="/assets/color/dark-theme.png" %}
+
+### Desaturated theme {% tag color="purple" %}Beta{%endtag%}
```

Comment:
the word `desaturated` is new and doesn't match with the code, except in the case of tile.

is desaturated a color palette? is it a variant on any of the existing color palettes?